src/main/java/uk/gov/hmcts/reform/et/syaapi/service/HubLinkService.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.et.syaapi.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.et.syaapi.enums.CaseEvent;
+import uk.gov.hmcts.reform.et.syaapi.helper.CaseDetailsConverter;
+import uk.gov.hmcts.reform.et.syaapi.helper.EmployeeObjectMapper;
+import uk.gov.hmcts.reform.et.syaapi.models.HubLinksStatusesRequest;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class HubLinkService {
+    private final CaseService caseService;
+    private final CaseDetailsConverter caseDetailsConverter;
+    private final FeatureToggleService featureToggleService;
+
+    /**
+     * Updates case data with hub link statuses {@link CaseDetails}.
+     *
+     * @param request       hub link status data
+     * @param authorization is used to find the {@link UserInfo} for request
+     * @return the associated {@link CaseDetails} if the case is created
+     */
+    public CaseDetails updateHubLinkStatuses(HubLinksStatusesRequest request, String authorization) {
+
+        if (featureToggleService.isCaseFlagsEnabled()) {
+            log.info("Case flags enabled - updating hub link status");
```

Comment:
Minor - maybe this log should match the log at L51 where it says calling rather than updating

Reply:
@hsjhita Made that change